aws-aurora-postgresql.yml

```diff
@@ -142,6 +153,9 @@ provision:
   - field_name: port
     type: integer
     details: The port number of the exposed database instance.
+  - field_name: require_ssl
```

Comment:
**question**: What is the purpose of this output? Is it going to be used in a binding? Is there a separate story for that?

Reply:
I replicated the same configuration applied in PostgreSQL. The property is expected to be exposed in the output result. Do you thing that we should change it?